internal/proto/data_coord.proto

```diff
@@ -694,3 +697,14 @@ message GetFlushStateRequest {
   string collection_name = 4;
   int64 collectionID = 5;
 }
+
+message ChannelOperations {
```

Comment:
Why not follow convention to call it request?

Reply:
yes,ok